packages/dataviews/src/dataviews-layouts/grid/index.tsx

```diff
@@ -227,9 +228,12 @@ function GridItem< Item >( {
 									direction="row"
 								>
 									<>
-										<FlexItem className="dataviews-view-grid__field-name">
-											{ field.header }
-										</FlexItem>
+										{ /* @ts-ignore */ }
+										<Tooltip text={ field.header }>
```

Comment:
I believe this should be using the `field.label` instead of `field.header` as the `Tooltip` internally expects a string? Then we can remove the `ts-ignore`.

Also, it should fix up the issue where field headers include other icons that don't play nicely with the `Tooltip`:

<img width="305" height="123" alt="image" src="https://github.com/user-attachments/assets/bd4ac0ba-c7f0-4b2d-a4ef-4f8de8d45b81" />